QuantumUNET/QNetworkServer.cs

```diff
@@ -1348,7 +1350,7 @@ private static void SendCrc(QNetworkConnection targetConnection)
 
 		private readonly HashSet<int> m_ExternalConnections;
 
-		public readonly ServerSimpleWrapper m_SimpleServerSimple;
+		private readonly ServerSimpleWrapper m_SimpleServerSimple;
```

Comment:
i think there were some other accessibility changes i made in QNet

Reply:
this is the only one that you changed with the variable syncer